RELEASING.md

```diff
@@ -6,11 +6,22 @@
 * Tag the release:
   * The tag must be called `vx.y.x`, e.g. `v3.0.0`.
   * To create the tag:
-    * https://github.com/scalameta/scalameta/releases/new
+    * https://github.com/scalameta/scalameta/releases/new (don't spend time
+      filling in the release notes - leave this field empty and edit it later;
```

Comment:
I propose we push a tag from locally first to trigger the travis job. That won't send an email notification to people watching the repo with empty release notes. Once the notes are complete you can "Publish release" on Github to existing tag and that won't trigger Travis job but it will send email notifications with the release notes and make the tag "verified"

Reply:
👍

Reply:
Done.